packages/fxa-settings/src/components/DeviceInfoBlock/mocks.tsx

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export const MOCK_IP_ADDRESS = 'XX.XX.XXX.XXX';
+export const MOCK_DEVICE_NAME = 'Ultron';
+export const MOCK_BROWSER_NAME = 'Firefox';
+export const MOCK_OS_NAME = 'macOS';
+export const MOCK_COUNTRY = 'Canada';
+export const MOCK_REGION = 'British Columbia';
+export const MOCK_CITY = 'Vancouver';
+
+export const MOCK_METADATA_WITH_LOCATION = {
```

Comment:
Casually noticed while reviewing some code below, but in other files we apply explicit types to these objects.
For example, in `packages/fxa-settings/src/pages/Pair/Supp/WaitForAuth/mocks.tsx` where we use the following:

```jsx
import { RemoteMetadata } from '../../../../lib/types';

...

export const MOCK_METADATA_WITH_LOCATION: RemoteMetadata = {
```